modules/core/src/views/view-manager.js

```diff
@@ -214,20 +214,16 @@ export default class ViewManager {
   }
 
   _createController(view) {
-    let controller = null;
-
-    if (view.controller) {
-      const Controller = view.controller;
-      controller = new Controller({
-        eventManager: this._eventManager,
-        viewState: this.getViewState(view.id),
-        // Set an internal callback that calls the prop callback if provided
-        onViewStateChange: this._onViewStateChange.bind(this, view.id),
-        onStateChange: this._eventCallbacks.onInteractiveStateChange
-      });
-    }
+    const controllerProps = Object.assign({}, view.controller, {
```

Comment:
Does this work if `view.controller` is `true`? 

We probably need test cases for the various signatures we support.